### PR TITLE
updated flag names

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: frontendservice/src/views/specificShop.vue [hunk 1]
```diff
@@ -127,16 +127,16 @@ export default {
...
                 } else {
                   self.allCookieList.splice(
                     self.allCookieList.indexOf(res.data.token.tokenId),
                     1
                   );
                   Cookie.set("tokenid", JSON.stringify(self.allCookieList));
-                  self.flag = false;
-                  self.statusFlag = true;
+                  self.isTokenInCookie = false;
+                  self.isTokenActive = false;
```
Comment (priytanvani) [inline frontendservice/src/views/specificShop.vue:139]: Didn't see the entire code, is this intensional?

File: frontendservice/src/views/specificShop.vue [hunk 2]
```diff
@@ -155,8 +155,8 @@ export default {
         Cookie.set("tokenid", JSON.stringify(self.allCookieList), {
           expires: 1
         });
-        self.statusFlag = false;
-        self.flag = true;
+        self.isTokenActive = true;
```
Comment (priytanvani) [inline frontendservice/src/views/specificShop.vue:158]: Didn't see the entire code, is this boolean flip intensional?

Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:158]: yes priyanka.

File: frontendservice/src/views/specificShop.vue (hunk 1 above)
Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:139]: yes priyanka.

**Pull request merged**